middleware/redirect.global.ts

```diff
@@ -0,0 +1,9 @@
+export default defineNuxtRouteMiddleware(
```

Comment:
I wanted to redirect to a "default" project if someone lands on `/`. Not sure if this is the best way to do that.

Very happy to change this for a better alternative.

Reply:
https://stackoverflow.com/a/75702298/5015955

Reply:
Perfect!